MockKitUI/View Controllers/IssueAlertTableViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  IssueAlertTableViewController.swift
+//  MockKitUI
+//
+//  Created by Rick Pasetto on 4/24/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import LoopKit
+import LoopKitUI
+import MockKit
+
+final class IssueAlertTableViewController: UITableViewController {
+  
+    let cgmManager: MockCGMManager
+
+    private enum AlertRow: Int, CaseIterable, CustomStringConvertible {
+        case immediate = 0
+        case delayed
+        case repeating
+        case issueLater
+        var description: String {
+            switch self {
+            case .immediate: return "Immediate"
+            case .delayed: return "Delayed 5 seconds"
+            case .repeating: return "Repeating every 8 seconds"
+            case .issueLater: return "10 seconds Later"
+            }
+        }
+        var trigger: DeviceAlert.Trigger {
```

Comment:
[nit] Can you add line breaks between each of the computed properties introduced in this scope?